contract_membership_delegated_partner/tests/test_delegated_partner_contract.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Tecnativa - David Vidal
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.tests import common
+
+
+class TestMembershipDelegateContract(common.SavepointCase):
```

Comment:
Can't you inherit from base contract test? What I usually do is to split a base test class in 2:

* One base class that inherits from `common.SavepointCase` which implements the `setUpClass` method.
* A second class that inherits from the previous base one, which implements the tests.

Then, on this module, inherit from base class and add only the records you need for the specific case. Well, for this specific case the setup is not very large, but imagine the same with other situations.

Reply:
@pedrobaeza Do you have an example? I can't figure out how to reach the contract tests route so I can inherit the class.

Reply:
`from odoo.addons.contract.tests.<test_file> import <test_class_name> `